frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -574,97 +665,83 @@ export function Experiment(): JSX.Element {
-                                    ?.slice(1)
-                                    .map((variant: MultivariateFlagVariant, idx: number) => (
-                                        <Col className="ml" key={idx}>
-                                            <div className="card-secondary">{variant.key} conversion rate</div>
-                                            <div className="text-center">
-                                                <b>--</b>{' '}
-                                                <Tag style={{ border: 'none' }} color={resultsTagColors[idx]}>
-                                                    <b>{variant.key}</b>
-                                                </Tag>
+                        {experimentResults ? (
+                            <Row justify="space-around">
+                                {experimentData.parameters.feature_flag_variants.map(
+                                    (variant: MultivariateFlagVariant, idx: number) => (
+                                        <Col key={idx}>
+                                            <div style={{ fontSize: 16 }}>
+                                                <b>{capitalizeFirstLetter(variant.key)}</b>
                                             </div>
+                                            {experimentInsightType === InsightType.FUNNELS
+                                                ? 'Conversion rate: '
+                                                : 'Count: '}
+                                            <b>
+                                                {experimentInsightType === InsightType.FUNNELS
+                                                    ? `${conversionRateForVariant(variant.key)}%`
+                                                    : countDataForVariant(variant.key)}
+                                            </b>
+                                            {experimentInsightType === InsightType.FUNNELS && (
+                                                <>
+                                                    <Progress
+                                                        percent={Number(conversionRateForVariant(variant.key))}
+                                                        size="small"
+                                                        showInfo={false}
+                                                        strokeColor={getSeriesColor(idx + 1)}
+                                                    />
+                                                    <div>
+                                                        Probability that this variant has higher conversion than other
+                                                        variants:{' '}
+                                                        <b>
+                                                            {(experimentResults.probability[variant.key] * 100).toFixed(
```

Comment:
I don't **really** know what probability that this variant has higher conversion than other variants really means compared to "conversion rate" / what the implication is. Let's see if users also feel that way?

<img width="426" alt="Screen Shot 2022-01-10 at 6 49 55 PM" src="https://user-images.githubusercontent.com/25164963/148857221-0d96c874-6ed3-4372-931b-7afcf3814eb7.png">

Reply:
Hmm, right, this text can be confusing.

What it actually means is: since this is a test, a sampling, the results you see here aren't necessarily true for the population. We measure this dissonance using the probability.

I _think_ we can sidestep this problem by being more implicit here (never thought I'd ever say this, wow).

`Probability that this variant is the best`